contrib/core-contract-tests/tests/bns/name_register.test.ts

```diff
@@ -366,8 +366,8 @@ describe("name revealing workflow", () => {
       Cl.tuple({
         owner: Cl.standardPrincipal(bob),
         ["zonefile-hash"]: Cl.bufferFromUtf8(cases[0].zonefile),
-        ["lease-ending-at"]: Cl.some(Cl.uint(16)),
-        ["lease-started-at"]: Cl.uint(6),
+        ["lease-ending-at"]: Cl.some(Cl.uint(17)),
+        ["lease-started-at"]: Cl.uint(7),
```

Comment:
Another approach would be to avoid hard cording the values here. wdyt?

```ts
    expect(nameResolve.result).toBeOk(
      Cl.tuple({
        owner: Cl.standardPrincipal(bob),
        ["zonefile-hash"]: Cl.bufferFromUtf8(cases[0].zonefile),
        ["lease-ending-at"]: Cl.some(Cl.uint(simnet.blockHeight + 10)),
        ["lease-started-at"]: Cl.uint(simnet.blockHeight),
      })
    );
```

Reply:
Yep, and that'll make the test-suite more robust, unless there's some implementation detail that I'm not currently aware of.

Reply:
@moodmosaic Do you want to update the PR before merge?

Reply:
@friedger yep, doing now 👍

Reply:
@friedger [done](https://github.com/stacks-network/stacks-core/pull/4087#event-11081929359).